### PR TITLE
3D radial density compensation and stacks of radial density compensation

### PR DESCRIPTION
**Pull request by zaccharieramzi**
This tackles #2 (don't know if it solves it though).

I tested the equivalent in my package (tfkbnufft) in TensorFlow and the density compensation worked perfectly.

### CODE REVIEW

File: torchkbnufft/mri/dcomp_calc.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import torch
 
 
-def calculate_radial_dcomp_pytorch(nufftob_forw, nufftob_back, ktraj):
+def calculate_radial_dcomp_pytorch(nufftob_forw, nufftob_back, ktraj, stacks=False):
```
Comment (mmuckley) [inline torchkbnufft/mri/dcomp_calc.py:7]: Shouldn't this default to True to maintain backwards compatibility?

**Review by mmuckley — CHANGES_REQUESTED** [review]
Thanks a bunch for looking into this. Only comment is we should default `stacks` to `True` to maintain backwards compatibility.

**Comment by mmuckley** [comment]
Note: if you find the default behavior undesirable, then we could also remove the default argument altogether to force users to be aware of the new functionality.

File: torchkbnufft/mri/dcomp_calc.py (hunk 1 above)
Comment (zaccharieramzi) [reply torchkbnufft/mri/dcomp_calc.py]: Done

**Comment by zaccharieramzi** [comment]
No no, I think surely you know how users were using it so far, so I will definitely change it back to `stacks=True` by default in 3D.

**Review by mmuckley — APPROVED** [review]

**Pull request merged**